src/Pelagos/Util/DatasetIndex.php

```diff
@@ -77,10 +77,12 @@ protected function buildQuery(array $termsFilters = array(), $text = null, $geoF
 
         if (!empty($text)) {
             $textQuery = new Query\BoolQuery();
-            $udiRegEx = '/\b([A-Z\d]{2}\.x\d\d\d\.\d\d\d:\d\d\d\d)\b/';
+            $udiRegEx = '/\b([a-zA-Z\d]{2}\.x\d\d\d\.\d\d\d[:.]\d\d\d\d)\b/';
```

Comment:
the 'x' is not  case sensitive. Recommend using /i in regex to make it case insensitive.

Reply:
Made that change